sql/src/test/resources/calcite/tests/window/no_grouping.sqlTest

```diff
@@ -3,7 +3,7 @@ type: "operatorValidation"
 sql: |
   SELECT
     m1,
-    COUNT(m1) OVER () cc
```

Comment:
Can we add a separate test if we are changing from COUNT to SUM

Reply:
...actually retaining the `COUNT` here was leading to a difference in the plans (and/or possibly to a bug) ; the goal in this patch was to enable the checks back - from the standpoint of this test the important thing is to run a query with at least 1 window function without any `group by` clauses

Reply:
We can add that with a failingTest annotation instead of the regular operatorValidation. That way we'll have the failing test in there so that once that bug gets address we can update it

Reply:
yep; that makes sense:
* added the new test as `no_grouping2.sqlTest`
* changed `no_grouping.sqlTest` to be `failingTest`